video_xblock/static/html/brightcove.html

```diff
@@ -18,7 +18,7 @@
         data-embed="default"
         data-application-id
         data-setup='{"playbackRates": [0.5, 1, 1.5, 2] }'
-        width="829"
+        width="100%"
         height="511"
```

Comment:
Yes, you right, but in this place it work good. We need height, because it's iframe. Video have good proportions, column with video have fixed width and it look fine.